### PR TITLE
Replace anvil with Geth

### PR DESCRIPTION
**Pull request by zsluedem**
Sadly anvil test node support debug_tracecall with javascript tracer. It would be better to switch to geth dev mode which support javascript tracer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ For more information: https://hackmd.io/@Vid201/aa-bundler-rust
 
 ## Prerequisites
 
-1. Ethereum execution client JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall). For production, you can use [Geth](https://github.com/ethereum/go-ethereum) or [Erigon](https://github.com/ledgerwatch/erigon). For testing purposes, you can use [Anvil](https://github.com/foundry-rs/foundry/tree/master/anvil#anvil), which provides enough functionalities of execution clients like Geth. To install Anvil, use [foundryup](https://getfoundry.sh/)
+1. Ethereum execution client JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall). For production, you can use [Geth](https://github.com/ethereum/go-ethereum) or [Erigon](https://github.com/ledgerwatch/erigon).
```
Comment (Vid201) [inline README.md:15]: We can add similar command as here: https://github.com/eth-infinitism/bundler#running-local-node

**Comment by zsluedem** [comment]
blocked by https://github.com/gakonst/ethers-rs/pull/2088 now

File: README.md (hunk 1 above)
Comment (zsluedem) [reply README.md]: Currently the Geth would start by the tests and you don't need to start by yourself. I think that's more convenient.

**Comment by zsluedem** [comment]
@Vid201 I marked this pr to be ready to review now. And I put the additional tracer stuff into https://github.com/Vid201/aa-bundler/issues/44 and I would make another pr for it. But I am not sure whether you got ongoing work related. I just want to notify you in case we have dup works.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -19,16 +19,13 @@ jobs:
       - name: Setup third party dependencies
         run: |
           make fetch-thirdparty
-      - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1
-        with:
-          version: nightly
       - name: Install solc
```
Comment (Vid201) [inline .github/workflows/ci.yml:22]: Install Geth and solc

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ For more information: https://hackmd.io/@Vid201/aa-bundler-rust
 
 ## Prerequisites
 
-1. Ethereum execution client JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall). For production, you can use [Geth](https://github.com/ethereum/go-ethereum) or [Erigon](https://github.com/ledgerwatch/erigon). For testing purposes, you can use [Anvil](https://github.com/foundry-rs/foundry/tree/master/anvil#anvil), which provides enough functionalities of execution clients like Geth. To install Anvil, use [foundryup](https://getfoundry.sh/)
+1. Ethereum execution client JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall). For production, you can use [Geth](https://github.com/ethereum/go-ethereum) or [Erigon](https://github.com/ledgerwatch/erigon). For testing, we are using Geth dev mode so you need to [install Geth](https://geth.ethereum.org/docs/getting-started/installing-geth) for running tests.
```
Comment (Vid201) [inline README.md:15]: Change to this: For testing, we are using Geth dev mode; so you need ...

Move install outside of square brackets: need to install [Geth] ...

**Review by Vid201 — CHANGES_REQUESTED** [review]

**Comment by Vid201** [comment]
> @Vid201 I marked this pr to be ready to review now. And I put the additional tracer stuff into #44 and I would make another pr for it. But I am not sure whether you got ongoing work related. I just want to notify you in case we have dup works.

That's fine, I sent you my progress on tracer on Discord.

This PR looks good, I left two small comments to change some text only.

**Review by Vid201 — APPROVED** [review]

**Pull request merged**